docs/UPGRADE_V1.6.0_TO_V1.7.0.md

```diff
@@ -0,0 +1,274 @@
+# Upgared Hub v1.6.0 to v1.7.0
+
+This doc defines the steps to follow for upgrading the Hub v1.6.0 to Hub v1.7.0.
+
+Navigate to `config` directoy in the root of the project and considering the `tekton-hub` namsespace.
+
+## Database Deploy
+
+Execute the below command
+
+```
+kubectl apply -f 00-init/ -n tekton-hub
+```
+
+This will create `tekton-hub` namespace, db deployment with `postgres` image, secret to save db credentials, pvc for the db and a ClusterIP service to expose the db internally.
+
+All resources are created in `tekton-hub` namespace.
+
+Wait till the pod comes in a running state
+
+```
+kubectl get pod -n tekton-hub -w
+```
+
+## Backup the Hub v1.6.0 Database
+
+Follow these steps sequentially to take the backup of Hub v1.6.0 DB:-
+
+Execute the below command to get the Hub v1.6.0 db pod
+
+```
+kubectl get pods -l app=db -n tekton-hub
+```
+
+Exec into the db pod
+
+```
+kubectl exec -it <db_pod_name> /bin/bash
+```
+
+Create a file within the pod and dump the Hub v1.6.0 db by running following following commands
+
+```
+touch /tmp/db.dump
```

Comment:
We don't need to create the file, running the pg_dump command will create one for it